src/status_im2/contexts/onboarding/common/intro/view.cljs

```diff
@@ -62,12 +64,12 @@
       :source (get-in carousels [@carousel-index :image])}]
     [quo/drawer-buttons
      {:top-card    {:on-press            (fn []
-                                           (rf/dispatch [:init-root :onboarding])
+                                           (rf/dispatch [:navigate-to :new-to-status])
```

Comment:
Instead of changing the root, we navigate, so we can `:navigate-back` from the `:new-to-status` screen